src/server/run-websocket-server.ts

```diff
@@ -69,6 +69,10 @@ export function runWebsocketServer(db: Knex, app: express.Application, augur: Au
             websocket.send(makeJsonRpcResponse(message.id, true));
           } else {
             dispatchJsonRpcRequest(db, message as JsonRpcRequest, augur, (err: Error|null, result?: any): void => {
+              if (websocket.readyState !== WebSocket.OPEN ) {
+                console.log("Client disconnected during request, ignoring response");
```

Comment:
This feels more like a `console.warn` to me...thoughts?

Reply:
yep! it's not normally a big deal, but it could be useful if you see a lot of them that something is up